packages/bundler-sdk-viem/test/helpers.ts

```diff
@@ -180,8 +180,11 @@ export const setupTestBundle = async <chain extends Chain = Chain>(
         ),
       ]);
 
-      for (const { balance } of balances)
-        expect(balance).toBeLessThanOrEqual(1n);
+      for (const { balance, adapter } of balances)
+        expect(
+          balance,
+          `balance of "${adapter}" for token "${token}"`,
+        ).toBeLessThanOrEqual(2n);
```

Comment:
```suggestion
        ).toBeLessThanOrEqual(5n);
```

That's the limit we defined in `finalizeBundle`